.github/workflows/commit-message.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Commit Message Check'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request_target:
```

Comment:
After review, we must keep only the `pull_request_target` event, because if we keep both of them, it will trigger additional check